dll/win32/shell32/lang/fr-FR.rc

```diff
@@ -948,40 +948,40 @@ BEGIN
-    IDS_ADVANCED_SHOW_INFO_TIP "Show pop-up description for folder and desktop items"
-
-    IDS_NEWEXT_ADVANCED_LEFT "<< Ad&vanced"
-    IDS_NEWEXT_ADVANCED_RIGHT "Ad&vanced >>"
-    IDS_NEWEXT_NEW "<New>"
-    IDS_NEWEXT_SPECIFY_EXT "You must specify an extension."
-    IDS_NEWEXT_ALREADY_ASSOC "Extension %s is already associated with File Type %s. Do you want to un-associate %s with %s and create a new File Type for it?"
-    IDS_NEWEXT_EXT_IN_USE "Extension is in use"
-
-    IDS_REMOVE_EXT "If you remove a registered file name extension, you will not be able to open files with this extension by double-clicking their icons.\n\nAre you sure you want to remove this extension?"
-    IDS_SPECIFY_ACTION "You must specify an action."
-    IDS_INVALID_PROGRAM "The specified program could not be found. Make sure the file name and path are correct."
-    IDS_REMOVE_ACTION "Are you sure you want to remove this action?"
-    IDS_ACTION_EXISTS "The action '%s' is already registered for this file type. Please enter a different name and try again."
-    IDS_EXE_FILTER "Programs\0*.exe\0All Files\0*.*\0"
-    IDS_EDITING_ACTION "Editing action for type: "
-    IDS_NO_ICONS "The file '%s' contains no icons.\n\nChoose an icon from the list or specify a different file."
-    IDS_FILE_NOT_FOUND "The file '%s' was not found."
+    IDS_ADVANCED_FOLDER "Fichiers et dossiers"
+    IDS_ADVANCED_NET_CRAWLER "Rechercher automatiquement les dossiers et imprimantes partagés"
+    IDS_ADVANCED_FOLDER_SIZE_TIP "Afficher les informations concernant la taille des fichiers dans les info-bulles"
+    IDS_ADVANCED_FRIENDLY_TREE "Afficher une vue simple des dossiers dans la liste des dossiers de l'Explorateur ReactOS"
+    IDS_ADVANCED_WEB_VIEW_BARRICADE "Affichier le contenu des dossiers système"
+    IDS_ADVANCED_SHOW_FULL_PATH_ADDRESS "Afficher le chemin complet dans la barre d'adresse"
+    IDS_ADVANCED_SHOW_FULL_PATH "Afficher le chemin complet dans la barre de titre"
+    IDS_ADVANCED_DISABLE_THUMB_CACHE "Ne pas mettre les miniatures en cache"
+    IDS_ADVANCED_HIDDEN "Fichiers et dossiers cachés"
+    IDS_ADVANCED_DONT_SHOW_HIDDEN "Ne pas afficher les fichiers et dossiers cachés"
+    IDS_ADVANCED_SHOW_HIDDEN "Afficher les fichiers et dossiers cachés"
+    IDS_ADVANCED_HIDE_FILE_EXT "Masquer les extensions des fichiers dont le type est connu"
+    IDS_ADVANCED_SUPER_HIDDEN "Masquer les fichiers protégés du système d'exploitation (recommandé)"
+    IDS_ADVANCED_DESKTOP_PROCESS "Ouvrir les fenêtres des dossiers dans un processus différent"
+    IDS_ADVANCED_CLASSIC_VIEW_STATE "Mémoriser les paramètres d'affichage de chaque dossier"
+    IDS_ADVANCED_PERSIST_BROWSERS "Restaurer les fenêtres de dossiers ouvertes lors de la prochaine ouverture de session"
+    IDS_ADVANCED_CONTROL_PANEL_IN_MY_COMPUTER "Afficher le Panneau de configuration dans le Poste de travail"
+    IDS_ADVANCED_SHOW_COMP_COLOR "Afficher les dossiers et les fichiers NTFS cryptés ou compressés en couleur"
```

Comment:
"chiffrés" is the right term in cryptography :p

Reply:
fixed